otseobande/maze-problem
# #162236449 implement board dimension dialog

**What does this PR do?**

This PR implements a dialog to receive board height and width from the user.

**Description of Task to be completed?**

- design components
- write tests

**How should this be manually tested?**

- Checkout to this branch
- Start the development server `$ yarn start`
- It should open a browser tab with the dimension dialg

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

- [#162236449](https://www.pivotaltracker.com/story/show/162236449)

**Screenshots (if appropriate)**

<img width="1440" alt="screen shot 2018-11-27 at 6 33 17 pm" src="https://user-images.githubusercontent.com/17123317/49099905-eeb78c00-f272-11e8-90f3-3ae90435ac25.png">


**Questions:**

N/A
